lib/sshutils/sftp/sftp.go

```diff
@@ -15,6 +15,8 @@ limitations under the License.
 */
 
 // Package sftp handles file transfers client-side via SFTP
+// path.Join is intentionally used over filepath.Join because SFTP
```

Comment:
I don't think this belongs in the package level doc. Why wasn't this placed down below where you actually use path.Join?

Reply:
I made a note in the original PR. `path.Join` has been used 3 or 4 other places in `sftp.go` already, the single `filepath.Join` call was a mistake. Rather than just including a comment explaining the use of the call once, or adding duplicate comments multiple places wherever `path.Join` is used, I thought it best to explain why it's used in a more central location.